web/web.go

```diff
@@ -55,15 +58,18 @@ func New(opts Options) *echo.Echo {
 	return e
 }
 
+// Start starts an HTTP server with custom address.
```

Comment:
`// StartServer start HTTP server with custom address.`
the name must begin with the name of the function by the standards of GoDoc

Reply:
but.. why not `StartServer HTTP with custom address`